R/integrate_sces.R

```diff
@@ -131,6 +142,7 @@ integrate_fastmnn <- function(merged_sce,
   # Perform integration
   integrated_sce <- batchelor::fastMNN(merged_sce,
                                        batch = colData(merged_sce)[,batch_column],
+                                       subset.row = hv_genes,
```

Comment:
Another question I have is whether we want to use `auto.merge` here?

Reply:
We've never done anything with that option, so I think we would want to test it out before using it automatically, but we could add it in as an option so we can more easily use it?

Reply:
I think we will want to test it, but we can do that in the script. I think we do want to use it, or some method of deciding merge order other than just the order we got things...